### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.9 es 4.51 (11 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#319] [Esercizi M-Z]
![es51](https://github.com/user-attachments/assets/ea94d9a1-739a-4fc3-9923-6b53c2105a28)

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/a0f085ce-d50a-49c8-a21d-54f391aee5aa)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/01f06cca-4896-4858-a13f-78208e7847ff)
